lib/auxiliars/helper.rb

```diff
@@ -74,6 +74,8 @@ def calculate_shipping_deadline(days, saturday, date = Date.today)
   # Converters
 
   def string_to_bool(string)
+    return false if string.nil?
+
     string.strip == 'S'
```

Comment:
💅🏾  Apenas para diminuir um pouco a sintaxe?
```suggestion
    string.strip == 'S' if string.present?
```

Reply:
Nesse caso eu prefiro o jeito que está.
Acredito ser mais clara a leitura com saída mais explicita de um fluxo alterantivo.
Justo?

Reply:
Sem problemas 😉